### ISSUE TITLE
Protein id not found in .gff file while present

### ISSUE THREAD

**Issue by rickbeeloo** [ncbi/amr#26]
**Problem**
I generated GFF files from PATRIC annotation files, this works for most genomes however now I came across an odd case.

AMRfinder gives me the erorr: `gff_check.cpp: Protein id "fig|1280.4850.peg.108" is not in the .gff-file`, however when I look at the .gff file it is present (see row 3):

```
accn|CP013616	PATRIC	CDS	103093	103575	.	+	0	ID=fig|1280.4850.peg.106;locus_tag=AUC48_00490;product=hypothetical protein
accn|CP013616	PATRIC	CDS	103781	104767	.	+	0	ID=fig|1280.4850.peg.107;locus_tag=AUC48_00495;product=Phosphatidylinositol-specific phospholipase C
accn|CP013616	PATRIC	CDS	104990	105421	.	+	0	ID=fig|1280.4850.peg.108;product=hypothetical protein
accn|CP013616	PATRIC	CDS	105411	105752	.	+	0	ID=fig|1280.4850.peg.109;product=hypothetical protein
accn|CP013616	PATRIC	CDS	105846	108083	.	+	0	ID=fig|1280.4850.peg.110;locus_tag=AUC48_00505;product=Transcriptional regulator%2C AraC family
```
See this [filebin](https://filebin.net/0oc1o5rkarvqmtjv) for the full FASTA and GFF files, then run `amrfinder -p 1280.4850.PATRIC.faa -g 1280.4850.PATRIC.gff` and it wil (in my case) display the above error.

**What I suspect**
It seems to occur for the identifiers not followed by `;locus_tag=` in my GFF file, however just adding a locus tag or copying one and adding it did not resolve this. 
I love this tool, so I hope we can find out the cause of this!

**Comment by evolarjun** [comment]
Hi Rickbeloo, 

Thanks for letting us know. I've never tried running AMRFinderPlus with PATRIC output, so it's great to hear it works (most of the time).

You are correct that `;locus_tag=` should not be required, but maybe something is not working the way we expect. Unfortunately I have a kid to attend to and I can't get to it tonight, but I'll take a look first thing tomorrow morning.

Thanks!

**Comment by rickbeeloo** [comment]
Hi Arjun,

Yeah sure! thanks for the quick reply - good luck attending :) 

It is not the actual PATRIC output, I transformed the .`PATRIC.features.tab`  (from [their FTP], .PATRIC.features.tab) file to gff3 in order to use it for AMRfinderPlus. Every time I remove the protein identifier producing the error it jumps to the next one lacking a locus tag, yet adding it does not solve it. So probably I miss something here that we can hopefully figure out tomorrow morning.

**Comment by vbrover** [comment]
https://github.com/ncbi/amr/wiki/Running-AMRFinderPlus#input-file-formats:
```
The value of the 'Name=' variable in the 9th field in the GFF must match the identifier in the protein FASTA file 
```
Can you try that?

**Comment by rickbeeloo** [comment]
I read that, I changed the `ID=` to `NAME=` in the 9th column, however, this does not make a difference. Notably, it only does not work in specific cases as I described above

**Comment by rickbeeloo** [comment]
I first changed it to `NAME=`... instead of `Name=`, now I changed it to `Name=` it does work! That was not that smart.. Still interesting that it did work in most cases except for those when the locus_tag was lacking. Anyway, thankyou!

**Comment by evolarjun** [comment]
Thanks Slava.  

I bolded the requirement for the `Name=` identifier in the help docs and added a section on known issues pointing to it. Hopefully people having trouble getting their GFF files to work will notice it more easily among all of our documentation.

**Comment by rickbeeloo** [comment]
Hi Arjun,

Note that the link to the example GFF ("test_prot.gff") at [https://github.com/ncbi/amr/wiki/Running-AMRFinderPlus#input-file-formats:](https://github.com/ncbi/amr/wiki/Running-AMRFinderPlus#input-file-formats:) is not working. Therefore I took a look at the GFF3 format at Ensembl where I saw the `ID=` tag in the 9th column for every instance (whereas this was not the case for `Name=`,) and wrongly assumed it was interchangeable with the `Name=` tag. So it may be useful to 1) fix the link to the example GFF or/and 2) perhaps provide a short example (like 3 lines of GFF) demonstrating the appropriate format.
